Error with the latest version, when specifying the duration flag

When I am trying to use styx to query results from Prometheus to CSV using the `-d` or `--duration` flag, I get the following errors:

`styx --prometheus http://${PROMETHEUS_IP}:${PROMETHEUS_PORT}  -d 1h '<some_metric>'`

```sh
 flag provided but not defined: -d
```

`styx --prometheus http://${PROMETHEUS_IP}:${PROMETHEUS_PORT}  --duration 1h '<some_metric>'`

```sh
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x2dbab8]

goroutine 1 [running]:
main.exportAction(0xc000188580)
        /root/go/pkg/mod/github.com/go-pluto/styx@v0.0.0-20230620093618-a4f4604c1052/main.go:101 +0x98
github.com/urfave/cli/v2.(*Command).Run(0xc0002b2840, 0xc000188580, {0xc000000240, 0x6, 0x6})
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.25.3/command.go:274 +0x76c
github.com/urfave/cli/v2.(*App).RunContext(0xc0002be000, {0x3f5200?, 0xc00001a070}, {0xc000000240, 0x6, 0x6})
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.25.3/app.go:332 +0x5dc
github.com/urfave/cli/v2.(*App).Run(...)
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.25.3/app.go:309
main.main()
        /root/go/pkg/mod/github.com/go-pluto/styx@v0.0.0-20230620093618-a4f4604c1052/main.go:90 +0x7d0
```

I am using version 1.20 of go (have also tried with 1.18)

Tested the commit `144d23073c05e9dfe4722e68515e8a09ff0ad249` and it works.

--- a/main.go
+++ b/main.go
@@ -26,6 +26,7 @@ func main() {
 			Name:   "start,s",
 			Usage:  "The start time to get timeseries from",
 			Layout: "2006-01-02T15:04:05",
+			Value: cli.NewTimestamp(time.Now().Truncate(time.Hour)),
 		},
 		&cli.DurationFlag{
 			Name:  "resolution,r",
@@ -60,6 +61,7 @@ func main() {
 				Name:   "start,s",
 				Usage:  "The start time to get timeseries from",
 				Layout: "2006-01-02T15:04:05",
+				Value: cli.NewTimestamp(time.Now().Truncate(time.Hour)),
 			},
 			&cli.DurationFlag{
 				Name:  "resolution,r",
@@ -89,6 +91,7 @@ func main() {
 				Name:   "start,s",
 				Usage:  "The start time to get timeseries from",
 				Layout: "2006-01-02T15:04:05",
+				Value: cli.NewTimestamp(time.Now().Truncate(time.Hour)),
 			},
 			&cli.DurationFlag{
 				Name:  "resolution,r",

--- a/main.go
+++ b/main.go
@@ -23,10 +23,11 @@ func main() {
 			Value: time.Hour,
 		},
 		&cli.TimestampFlag{
-			Name:   "start,s",
-			Usage:  "The start time to get timeseries from",
-			Layout: "2006-01-02T15:04:05",
-			Value: cli.NewTimestamp(time.Now().Truncate(time.Hour)),
+			Name:    "start,s",
+			Usage:   "The start time to get timeseries from",
+			Layout:  "2006-01-02T15:04:05",
+			Value:    cli.NewTimestamp(time.Now().Truncate(time.Hour)),
+			Timezone: time.Local,
 		},
 		&cli.DurationFlag{
 			Name:  "resolution,r",
@@ -58,10 +59,11 @@ func main() {
 				Value: time.Hour,
 			},
 			&cli.TimestampFlag{
-				Name:   "start,s",
-				Usage:  "The start time to get timeseries from",
-				Layout: "2006-01-02T15:04:05",
-				Value: cli.NewTimestamp(time.Now().Truncate(time.Hour)),
+				Name:     "start,s",
+				Usage:    "The start time to get timeseries from",
+				Layout:   "2006-01-02T15:04:05",
+				Value:    cli.NewTimestamp(time.Now().Truncate(time.Hour)),
+				Timezone: time.Local,
 			},
 			&cli.DurationFlag{
 				Name:  "resolution,r",
@@ -92,6 +94,7 @@ func main() {
 				Usage:  "The start time to get timeseries from",
 				Layout: "2006-01-02T15:04:05",
 				Value: cli.NewTimestamp(time.Now().Truncate(time.Hour)),
+				Timezone: time.Local,
 			},
 			&cli.DurationFlag{
 				Name:  "resolution,r",

